docker/docker-compose.yml

```diff
@@ -4,47 +4,47 @@ services:
   db:
     image: postgres:9.5.3
     volumes:
-      - $VAGRANT_DATA_DIR../data/pgdata:/var/lib/postgresql/data/pgdata
-    environment:
-      PGDATA: /var/lib/postgresql/data/pgdata
-      POSTGRES_USER: listenbrainz
-      POSTGRES_PASSWD: listenbrainz
-      POSTGRES_DB: listenbrainz
+      - ./data/pgdata:/var/lib/postgresql
+    ports:
+      - "15432:5432"
 
   redis:
     image: redis:3.2.1
+    volumes:
+      - ./data/redis:/data
 
-  influxdb:
+  influx:
     image: influxdb:1.0.0
     volumes:
-      - $VAGRANT_DATA_DIR../data/influxdb:/data
-    restart: always
+      - ./data/influxdb:/var/lib/influxdb
+    environment:
+      INFLUXDB_REPORTING_DISABLED: 'true'
+      INFLUXDB_META_LOGGING_ENABLED: 'false'
+      INFLUXDB_DATA_QUERY_LOG_ENABLED: 'false'
+      INFLUXDB_DATA_WAL_LOGGING_ENABLED: 'false'
+      INFLUXDB_DATA_DATA_LOGGING_ENABLED: 'false'
+      INFLUXDB_HTTP_LOG_ENABLED: 'true'
+      INFLUXDB_CONTINUOUS_QUERIES_LOG_ENABLED: 'false'
 
+    ports:
+      - "8086:8086"
+      - "8083:8083"
 
   web:
     build:
       context: ..
       dockerfile: ./docker/Dockerfile
+    command: uwsgi /code/listenbrainz/docker/uwsgi.ini
```

Comment:
Does this mean that during development we'll be running with nginx/uwsgi? Do you have a reason why we do this instead of `manage.py runserver`?
I know I normally go on about docker letting us run production setup in devlopment, but I think this is a bit over the top.

If we do decide to go with this, we should configure uwsgi to only use one thread, and to enable the code reloader.